.idea/.idea.magic-unity.dir/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does the ./idea folder need to be added to .gitignore?

Reply:
Good catch. How did this got picked up by git even I have .idea in gitignore